### DISCUSSION TITLE
dynamic playlist

### DISCUSSION THREAD

**Discussion by dataguy73301** [smplayer-dev/smplayer#752] [Q&A]
Because I add and delete video files frequently is there a method to create a dynamic smplayer playlist .m3u?  I'm using Windows 11 and the latest version of SMPlayer.

Thank you in advance for any suggestions.

**Comment by Raza-0007** [comment]
The way I enable a dynamic playlist is by using the following option:

"Options>Preferences>Playlist>Adding files>Add files to the playlist automatically>Add files from the folder:Video and audio files"

After this all I need to do is to play any file inside a folder, and it populates the rest of the files in the playlist for me. So this way it does not matter if the files inside the folder are changing, the playlist updates itself on every playback automatically.

Of course this works best if the files inside the folder are already sorted by file numbers/track numbers etc.

You can tinker with the other three options in the drop down menu on the setting to see which one works the best for you.

**Comment by aris95** [comment]
Is there posssible "Auto add similar files in folder to (temporary) playlist? I'm using linux version
